build/docker/start_kmesh.sh

```diff
@@ -4,8 +4,7 @@
 
 lsmod | grep kmesh > /dev/null
 if [ $? -ne 0 ] && [ -f kmesh.ko ]; then
-        mkdir -p /lib/modules/kmesh
-        cp kmesh.ko /lib/modules/kmesh
+	cp kmesh.ko /lib/modules/$(uname -r)
```

Comment:
Why do I need to make this change, and what's wrong with copying to /lib/modules/kmesh?

Reply:
The modprobe kmesh command can search for and load ko only in /lib/modules/$(uname -r).
In the past, a Symbolic Link of kmesh.ko is created in the directory when the docker is made locally. Therefore, this problem is not found,
Actually, ko cannot be loaded when the kmesh image is directly pulled.